benchmarks/bvh_driver/bvh_driver.cpp

```diff
@@ -51,9 +54,12 @@ struct Spec
     return s;
   }
 
-  std::string create_label_radius_search(std::string const &tree_name) const
+  std::string create_label_radius_search(std::string const &tree_name,
+                                         std::string const &flavor = "") const
```

Comment:
I would also get rid of the default argument here.

Reply:
No, there is a place that calls it with default argument.